spyder/app/tests/test_mainwindow.py

```diff
@@ -759,10 +761,23 @@ def test_dedicated_consoles(main_window, qtbot):
     text = control.toPlainText()
     assert ('runfile' in text) and not ('Python' in text or 'IPython' in text)
 
-    # --- Clean namespace after re-execution ---
+    # --- Clean retained after re-execution ---
```

Comment:
Maybe this comment could be clearer writting something like `Check namespace retention after re-execution`?

Reply:
I agree with @dalthviz's comment too. Otherwise this PR looks good to me.